### ISSUE TITLE
[Bug][SF][2.6.0][FlexibleColumnLayout][A11y][Expected reply Time: ASAP] Keyboard focus not visible on separator in High Contrast Mode

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents-react#6865]
### Describe the bug

<img width="1137" alt="Image" src="https://github.com/user-attachments/assets/ed2f6b03-7124-4d59-9ba0-85da52ac2b42" />

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-flexiblecolumnlayout--docs&args=disableResizing:!true&globals=theme:sap_horizon_hcb

### Reproduction steps

Press tab to focus on sequence.

### Expected Behaviour

The keyboard focus should be visible on the separator in all modes.

### UI5 Web Components for React Version

2.6.0

### UI5 Web Components Version

2.6.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
